### DISCUSSION TITLE
Is it possible to manually do a sync request with OneDriveGUI running?

### DISCUSSION THREAD

**Discussion by mistune** [abraunegg/onedrive#3009] [Q&A] [answered]
Initial setup with the client with OneDriveGUI after migrating away from another OneDrive client. I'm done setting up the onedrive client from OneDriveGUI, but have not configured webhooks yet. Is there a way to manually do a sync with OneDriveGUI running?

I'm currently behind a CGNAT (I've read the Microsoft Graph API subscription requirement, and it said they need a reachable URL for this) so from my limited experience on this topic, it will take a while for me to figure out how to do this.

For the meantime, I would like to do the sync requests manually.

**Comment by abraunegg** [comment, marked as answer]
@mistune 
> Is there a way to manually do a sync with OneDriveGUI running?

This is not possible.

The OneDriveGUI is running a systemd service using your account configuration. You must first stop this systemd service / account configuration to then do a manual sync using that account configuration.

There is no work around / alternative approach.
